docs/reference/spec/host-capabilities/06-kubernetes.md

```diff
@@ -14,35 +14,46 @@ doc-topic:
   <link rel="canonical" href="https://docs.kubewarden.io/reference/spec/host-capabilities/kubernetes"/>
 </head>
 
-Kubewarden context aware policies require access to resources from the Kubernetes cluster where they are running.
-For that, the Kubewarden SDKs expose functions that use the waPC communication protocol to talk with the host system asking for data about the cluster.
+Kubewarden context aware policies require access to resources from the
+Kubernetes cluster where they are running. For that, the Kubewarden SDKs expose
+functions that use the waPC communication protocol to talk with the host system
+asking for data about the cluster.
 
 ## waPC protocol contract
 
-Internally, the SDKs rely on these functions exposed by the policy host environment:
+Internally, the SDKs rely on these functions exposed by the policy host
+environment:
 
-- `list_resources_by_namespace` : Given a resource type and a namespace, list all the resources of that type that are defined in it.
-  This cannot be used to list cluster-wide resources, like `Namespace`.
-- `list_resources_all`: Given a resource type, list all the resources of that type that are defined inside the whole cluster.
-  This can be used to list cluster-wide resources, like `Namespace`.
-- `get_resource`: Find the exact resource identified by the given resource type, given name and an optional namespace identifier.
+- `list_resources_by_namespace` : Given a resource type and a namespace, list
+  all the resources of that type that are defined in it. This cannot be used to
+  list cluster-wide resources, like `Namespace`.
+- `list_resources_all`: Given a resource type, list all the resources of that
+  type that are defined inside the whole cluster. This can be used to list
+  cluster-wide resources, like `Namespace`.
+- `get_resource`: Find the exact resource identified by the given resource
+  type, given name and an optional namespace identifier.
+- `can_i`: Allow policy authors to send `SubjectAccessReview` object to Kubernetes
+  authorization API to verify user permissions
```

Comment:
```suggestion
  authorization API to verify user permissions.
```

Reply:
I've updated the suggestion made by @viccuad to include also a link to the authorization docs of k8s. This is kinda of a niche API